src/game/SpellAuras.cpp

```diff
@@ -5724,6 +5724,19 @@ void Aura::PeriodicTick()
                         }
                         break;
                     }
+                    case 46771: {                               // Flame Sear
+                        if (target->GetTypeId() == TYPEID_PLAYER) {
+                            if (!target->HasAura(45343) &&
+                                !target->HasAura(47300)) {          // No Dark Flame applied. There are two auras
+                                if (m_periodicTick % 4 == 1) {      // Only doze the aura every 2 ticks (should maybe be one second cooldown)
+                                    target->CastSpell(target, 45348,
```

Comment:
The boss mecanic is the following:

- One fire lady does fire damage and applies fire debuff when it ticks
- One shadow lady does shadow damage and applies shadow debuff when it ticks
- When someone has a fire or a shadow debuff and take damage from the opposite school, he gains 3 seconds of immunity for all debuff applications

I assume one aura is triggered when switching from fire to shadow and the other the other why around.

This test seems legit to me